server/model.py

```diff
@@ -9,10 +8,35 @@ class Users(db.Model):
     email = db.Column(db.String(64), unique=True, index=True)
     username = db.Column(db.String(64), unique=True, index=True)
     password_hash = db.Column(db.String(128))
+    eventType = db.relationship("EventTypes", backref="user")
 
-    def __init__(self, name, email, username, password_hash):
+    def __init__(self, name, email, username, password_hash, eventType):
```

Comment:
Actually, eventType shouldn't be in constructor, same for appointment below,. Will make commit after everything else is good.